build.sbt

```diff
@@ -42,21 +29,15 @@ lazy val microservice =
       scalacOptions += "-Wconf:src=routes/.*:s"
     )
     .configs(IntegrationTest)
-    .settings(inConfig(IntegrationTest)(Defaults.itSettings): _*)
+    .settings(DefaultBuildSettings.integrationTestSettings())
     .settings(
-      majorVersion := 3,
       IntegrationTest / Keys.fork := false,
       IntegrationTest / unmanagedSourceDirectories := (IntegrationTest / baseDirectory)(base => Seq(base / "it")).value,
       addTestReportOption(IntegrationTest, "int-test-reports"),
       IntegrationTest / testGrouping := oneForkedJvmPerTest((IntegrationTest / definedTests).value),
-      IntegrationTest / parallelExecution := false
+      IntegrationTest / parallelExecution := false,
+      Test / parallelExecution := false
     )
```

Comment:
Can all these be removed please as integrationTestSettings has been added. They are brought in by sbt-settings [here](https://github.com/hmrc/sbt-settings/blob/main/src/main/scala/uk/gov/hmrc/DefaultBuildSettings.scala#L93-L124) via enabling sbt-auto-build plugin. Thanks

Reply:
In SbtAutoBuildPlugin in sbt-auto-build, it can be seen [here](https://github.com/hmrc/sbt-auto-build/blob/7ea42a0d49d22ad33864bba3b4f8e612c14a98ff/src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala#L51-L53) where scalaSettings and defaultSettings are brought in and can be seen from the logged message when you run the tests which is why they can be removed.
<img width="496" alt="Screenshot 2023-07-12 at 17 05 10" src="https://github.com/hmrc/tax-history/assets/63714619/f4484937-04f8-46a1-b4bf-a5143840735e">

Reply:
In SbtAutoBuildPlugin in sbt-auto-build, it can be seen [here](https://github.com/hmrc/sbt-auto-build/blob/7ea42a0d49d22ad33864bba3b4f8e612c14a98ff/src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala#L51-L53) where scalaSettings and defaultSettings are brought in and can be seen from the logged message when you run the tests which is why they can be removed.
<img width="496" alt="Screenshot 2023-07-12 at 17 05 10" src="https://github.com/hmrc/tax-history/assets/63714619/f4484937-04f8-46a1-b4bf-a5143840735e">